packages/jstz-types/index.d.ts

```diff
@@ -85,7 +85,7 @@ declare interface Request extends Body {
 
 declare var Request: {
   readonly prototype: Request;
-  new (input: RequestInfo, init?: RequestInit): Request;
+  new(input: RequestInfo, init?: RequestInit): Request;
 };
```

Comment:
What is with the removal of the ` `

Reply:
I get autoformat on files.. seems like this one got formatted, should I revert ? are you using a different formatter ?

Reply:
Perhaps your prettier version differs from ours